README.md

```diff
@@ -26,11 +26,18 @@ $ cat /path/to/key | base64
 # then past it into data.tls.key
 ```
 
+Edit **MongoDB_HA/storage.yaml** to replace the **google cloud SSD** by solution that you want.
```

Comment:
You are missing a word before the **solution** part.

Reply:
Added word "the"